enhancements/ingress/gateway-api-crd-life-cycle-management.md

```diff
@@ -0,0 +1,470 @@
+>   disruption](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/). Any exception to
+>   this should be identified and discussed here.
+> - Micro version upgrades - users should be able to skip forward versions within a
+>   minor release stream without being required to pass through intermediate
+>   versions - i.e. `x.y.N->x.y.N+2` should work without requiring `x.y.N->x.y.N+1`
+>   as an intermediate step.
+> - Minor version upgrades - you only need to support `x.N->x.N+1` upgrade
+>   steps. So, for example, it is acceptable to require a user running 4.3 to
+>   upgrade to 4.5 with a `4.3->4.4` step followed by a `4.4->4.5` step.
+> - While an upgrade is in progress, new component versions should
+>   continue to operate correctly in concert with older component
+>   versions (aka "version skew"). For example, if a node is down, and
+>   an operator is rolling out a daemonset, the old and new daemonset
+>   pods must continue to work correctly even while the cluster remains
+>   in this partially upgraded state for some time.
+>
+> Downgrade expectations:
+> - If an `N->N+1` upgrade fails mid-way through, or if the `N+1` cluster is
+>   misbehaving, it should be possible for the user to rollback to `N`. It is
+>   acceptable to require some documented manual steps in order to fully restore
+>   the downgraded cluster to its previous state. Examples of acceptable steps
+>   include:
+>   - Deleting any CVO-managed resources added by the new version. The
+>     CVO does not currently delete resources that no longer exist in
+>     the target version.
+
+## Operational Aspects of API Extensions
+
+Other products and components that have Gateway API support will now be able to
+consistently know that Gateway API will already be present on the cluster, and
+which version will be present given the version of OpenShift. There will no
+longer be a need for them to document having their users deploy the CRDs
+manually or do any management themselves that could conflict.
+
+We are already aware of several projects which utilize Gateway API including
+(but not limited to):
+
+* OpenShift Service Mesh
+* Kuadrant
+* OpenShift AI Serving
+
+We will coordinate with these projects and others from release to release on
+their needs related to Gateway API version support. We expect over time that
+more flexibility with the version will eventually be needed, and we anticipate
+adding ranges of support instead of specific versions to accomodate this.
+
+## Support Procedures
+
+### Conflicting CRDs
+
+The pre-upgrade checks should eliminate any problems with CRD conflicts.
+However it is always _technically possible_ for the admin to force through both
+the pre-upgrade check AND the admin gate. If they do this the CIO will detect
+the mismatching schema and report a `Degraded` status condition with status
+`True` and a message explaining the problem.
```

Comment:
Did you mean `Upgradeable=false` or I'm missing something?

Reply:
It is my understanding that we will set the condition named `Degraded` and set it (itself) to `True`.

Reply:
Yes. Let me know though if you think any further clarification is needed!

Reply:
Sounds good, I added further clarification in the notes for the workflow in e66cab039588f180762adfe9e8d413745644d3f1, LMKWYT?